app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicsListAdapter.kt

```diff
@@ -11,11 +10,12 @@ import com.worldsnas.starplayer.model.Music
 
 class MusicsListAdapter :
     ListAdapter<Music, MusicListItemViewHolder>(DiffCallback) {
+    var onItemClick: ((Music) -> Unit)? = null
```

Comment:
inro inja mishe to constructor adapter daryaft kard
va mitone nullable ham nabashe

Reply:
kodom behtare , goftm dast be constructor nazanam shayad behtar bashe